app/components/Send/SendPanel/SendSuccess/index.jsx

```diff
@@ -23,6 +24,14 @@ export default class SendSuccess extends React.Component<Props> {
 
   render() {
     const { sendRowDetails, txid } = this.props
+    // normalize tx data for Transaction
+    const transactions = sendRowDetails.map(row => ({
+      iconType: 'SEND',
```

Comment:
I really like this approach 😄- now that iconType is being used across the application do you mind adding an exported enum  to `constants.js` for these constant values?